README.md

```diff
@@ -1,53 +1,93 @@
 # Template Extension Specification
 
-- **Title:** Template
-- **Identifier:** <https://stac-extensions.github.io/template/v1.0.0/schema.json>
-- **Field Name Prefix:** template
+- **Title:** Classification
+- **Identifier:** <https://stac-extensions.github.io/classification/v1.0.0/schema.json>
+- **Field Name Prefix:** classification
 - **Scope:** Item, Collection
 - **Extension [Maturity Classification](https://github.com/radiantearth/stac-spec/tree/master/extensions/README.md#extension-maturity):** Proposal
-- **Owner**: @your-gh-handles @person2
+- **Owner**: @drwelby @mmohr
+
+This document explains the Classification Extension to the [SpatioTemporal Asset Catalog](https://github.com/radiantearth/stac-spec) (STAC) specification.
+
+Classification stores metadata that clarifies the values within a dataset. Common uses would be:
+
+- Describing classes of data, and the values belonging to the class
+- The reverse of the above, as a lookup table mapping values to classes
 
-This document explains the Template Extension to the [SpatioTemporal Asset Catalog](https://github.com/radiantearth/stac-spec) (STAC) specification.
-This is the place to add a short introduction.
 
 - Examples:
-  - [Item example](examples/item.json): Shows the basic usage of the extension in a STAC Item
+  - [Asset example](examples/item.json): Shows the basic usage of the extension in a STAC Item
   - [Collection example](examples/collection.json): Shows the basic usage of the extension in a STAC Collection
 - [JSON Schema](json-schema/schema.json)
 - [Changelog](./CHANGELOG.md)
 
-## Item Properties and Collection Fields
+## Asset Properties 
+
+For single-band rasters
+
+## Raster Band (Raster Extension)
+
+For multiband rasters
+
+## Table Column (Table Extension)
+
+For tabular or vector datasets
+
+## Collection Fields
+
+In `summaries` field
```

Comment:
Is there some content missing here? I'm not quite sure what this is trying to express.

Reply:
sort of a checklist